### ISSUE TITLE
Build Fails due to wrong _dir

### ISSUE THREAD

**Issue by jhbruhn** [ros-melodic-arch/ros-melodic-gazebo-dev#7]
Since the last update, the build fails:
```
grep: /home/jhbruhn/.cache/yay/ros-melodic-gazebo-dev/src/gazebo_ros_pkgs-2.8.6/gazebo-dev: Datei oder Verzeichnis nicht gefunden
CMake Error: The source directory "/home/jhbruhn/.cache/yay/ros-melodic-gazebo-dev/src/gazebo_ros_pkgs-2.8.6/gazebo-dev" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
```

apparently the `_dir` value in the PKGBUILD is incorrect, it should be `gazebo_dev` instead of `gazebo-dev`

**Comment by acxz** [comment]
can confirm, im a bit busy this weekend and if you get around to fixing this before us we will go ahead and merge your PR

**Comment by acxz** [comment]
Should be fixed with 496f776

**Comment by acxz** [comment]
synced
